packages/kuma-gui/src/app/kuma/debug.ts

```diff
@@ -10,6 +10,21 @@ const $ = {
 }
 
 export const services = (app: Record<string, Token>): ServiceDefinition[] => [
+  [token('kuma.locales.debug'), {
+    service: () => {
+      return {
+        common: {
+          product: {
+            docs: 'https://kuma.io/docs/dev',
+          },
+        },
+      }
```

Comment:
This service definition file is sourced by Cypress, which doesn't have a loader configured for `.yaml` files, so this means I can't use `import {...} from '....yaml'` here.

I figured I could just not use yaml here seeing as its one little value to get around this.